pyproject.toml

```diff
@@ -202,45 +201,33 @@ manylinux-aarch64-image = "manylinux_2_28"
 musllinux-x86_64-image = "musllinux_1_2"
 musllinux-aarch64-image = "musllinux_1_2"
 
-[tool.cibuildwheel.pyodide]
-before-test = "pip install -r {project}/requirements/emscripten_test_requirements.txt"
-# Pyodide ensures that the wheels are already repaired by auditwheel-emscripten
-repair-wheel-command = ""
-test-command = "python -m pytest --pyargs numpy -m 'not slow'"
-
-[tool.cibuildwheel.pyodide.config-settings]
-build-dir = "build"
-setup-args = ["--cross-file=$PWD/tools/ci/emscripten/emscripten.meson.cross", "-Dblas=none", "-Dlapack=none"]
-
 [tool.cibuildwheel.linux.environment]
-# RUNNER_OS is a GitHub Actions specific env var; define it here so it works on Cirrus CI too
+# RUNNER_OS is a GitHub Actions specific env var; define it here so it's
+# defined when running cibuildwheel locally
 RUNNER_OS="Linux"
 # /project will be the $PWD equivalent inside the docker used to build the wheel
 PKG_CONFIG_PATH="/project/.openblas"
-LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/project/.openblas/lib"
-
-[tool.cibuildwheel.macos]
```

Comment:
So these entries are being removed because we only need them in numpy-release now?

Reply:
No we still have macOS wheel builds here in `wheels.yml`. I actually improved how this bundling works, so messing with `DYLD_LIBRARY_PATH` is no longer necessary. If you look at https://github.com/numpy/numpy-release/blob/main/cibuildwheel.toml, you won't find a `macos` entry there either.